README.md

```diff
@@ -1,280 +1,231 @@
-script. The script will setup everything that is necessary. This includes the
-_Parity_ client, an automated update service and more. Therefore the script will
-interact with the user to request missing information. The script can be called
-multiple times without problems. It checks what is already setup and what is
-left. At least all processes get updated and restarted on each run. The _Parity_
-client will restart automatically on failures.
+There are multiple ways to set each of these up. The most straightforward one by far is via our interactive quickstart
+script. Finer control can be achieved by starting the components individually as docker containers. Finally, it is also
+possible to avoid docker altogether and run everything directly on the host machine.
 
-```sh
-$ wget -O quickstart.sh https://github.com/trustlines-protocol/blockchain/raw/master/quickstart.sh && bash quickstart.sh
-```
-
-Follow the instructions to insert your password. If you want to restart the node or want to make sure it runs on the most recent version, just
-rerun the script.
-
----
+Before starting the installation process, please have a look at the system requirements and the note on backups.
 
-### System Time
+### System Requirements
 
-Due to the way the block validation works and is synchronized, it is essential to make sure your host system has the correct time configuration. On recent Ubuntu systems, for example, this should already be the case. You can check the settings on such systems using `timedatectl`.
+Based on the experiences we have had on our long-running testnet, we recommend at least 2GB of memory and 10GB of SSD
+storage.
 
-On other operating systems you should check if your time is synchronized with an [NTP server](https://www.pool.ntp.org/).
+Validators should make sure their node has a high uptime: Otherwise, they miss out on potential revenue and harm the
+network by increasing average block times.
 
----
+For block validation and creation, it is essential to make sure your host system has the correct time configuration. On
+most systems, this should be the case by default. You can check the settings with `timedatectl` (look for
+`"System clock synchronized: yes"`).
 
-### With Docker
+For the quickstart and docker installation modes, [docker](https://docker.com) needs to be installed and configured.
+Please refer to the official documentation and make sure your user is added to the `docker` user group if you cannot
+access root permissions to run containers.
 
-The following instructions explain how to start a local peer with the _Docker_ image. In fact it use a pre-configured [Parity
-Ethereum](https://www.parity.io/) client, combined with a set-up wrapper, to make connecting as easy as possible. The image is prepared to be
-used in three different scenarios: as observer, to participate and connect as validator.
+### Backups
 
-#### Pre-Requisites
+For validators it is crucial to safely back up their private key.
```

Comment:
I moved it up explicitly to make it more visible and because the reading paths diverge depending on the installation method, so few people will read until the end.

Since validators participated in the auction with the same key already, they should have backups at this point and most people won't have to use `scp`.

Reply:
Okay, I'm fine with that opinion. The key is the only thing that actually matters.